### PR TITLE
odo add binding documentation

### PR DESCRIPTION
**Pull request by valaparthvi**
Signed-off-by: Parthvi Vala <pvala@redhat.com>

**What type of PR is this:**

/kind documentation

**What does this PR do / why we need it:**
This PR adds documentation for `odo add binding`.

**Which issue(s) this PR fixes:**

Fixes part of #5627 

**PR acceptance criteria:**

- [ ] Unit test 

- [ ] Integration test 

- [x] Documentation 

**How to test changes / Special notes to the reviewer:**

### CODE REVIEW

**Comment by valaparthvi** [comment]
/hold
Until #5641 is merged.

File: docs/website/versioned_docs/version-3.0.0/command-reference/add-binding.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+### Formats supported by the `--service` flag
+The `--service` flag supports the following formats to specify the service name:
+* `<name>`
+* `<name>.<kind>`
+* `<name>.<kind>.<apigroup>`
+* `<name>/<kind>`
+* `<name>/<kind>.<apigroup>`
+
+The above formats are helpful when multiple services with the same name exists on the cluster.
```
Comment (feloy) [inline docs/website/versioned_docs/version-3.0.0/command-reference/add-binding.md:48]:
```suggestion
The above formats are helpful when multiple services with the same name exist on the cluster.
```

File: docs/website/versioned_docs/version-3.0.0/command-reference/add-binding.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+#### Examples - 
+```shell
+# Add binding between a service named 'myservice',
+# and the component present in the working directory
+odo add binding --service myservice --name myRedisService
+
+# Add binding between service named 'myservice' of kind 'Redis', and APIGroup 'redis.redis.opstreelab.in',
+# and the component present in the working directory 
+odo add binding --service myservice/Redis.redis.redis.opstreelab.in --name myRedisService
+odo add binding --service myservice.Redis.redis.redis.opstreelab.in --name myRedisService
```
Comment (feloy) [inline docs/website/versioned_docs/version-3.0.0/command-reference/add-binding.md:59]:
```suggestion
```

File: docs/website/versioned_docs/version-3.0.0/command-reference/add-binding.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+# Add binding between service named 'myservice' of kind 'Redis',
+# and the component present in the working directory
+odo add binding --service myservice/Redis --name myRedisService
+odo add binding --service myservice.Redis --name myRedisService
```
Comment (feloy) [inline docs/website/versioned_docs/version-3.0.0/command-reference/add-binding.md:64]:
```suggestion
```

**Review by feloy — CHANGES_REQUESTED** [review]

File: docs/website/versioned_docs/version-3.0.0/command-reference/add-binding.md (hunk 2 above)
Comment (feloy) [reply docs/website/versioned_docs/version-3.0.0/command-reference/add-binding.md:59]: Oh sorry, I didn't see the difference between the two lines. 
Perhaps you can highlight the difference in some way, with some "note the different notations for the service name"?

**Review by feloy — APPROVED** [review]

**Pull request merged**